src/main/java/io/curity/authenticator/netid/client/NetIdAccessAuthenticationAttributes.java

```diff
@@ -103,7 +103,7 @@ private static SubjectAttributes getSubjectAttributes(ResultCollect result)
     private static void handleSubject(UserInfoType userInfo, List<Attribute> subjectAttributes)
     {
         @Nullable String personalNumber = userInfo.getPersonalNumber();
-        @Nullable String userId = userInfo.getUserId();
```

Comment:
The `userId` is still present in the WSDL so we could fall back to this one also. We can also fall back to the `requestedUserId` field, as it seems that it is the only one that will be in the response for sure. Any thoughts?

Reply:
We need to find a subject to use, and as long as all the attributes we find is available in the result the admin can change to the one they want using an action. To not break previous functinoality, I suggest we do a prio-list:
`personalNumber -> userId -> uniqueUserName -> requestedUserId`, and then put all four of them in the result. Wdyt?

Reply:
Ok, so we populate the subject according to the prio-list, then add all four as subject parameters to the result, right? It seems like a good approach.